zingolib/src/commands.rs

```diff
@@ -1379,48 +1353,6 @@ impl Command for NewAddressCommand {
     }
 }
 
-struct NotesCommand {}
```

Comment:
there is still code in zingo-mobile thats calls this command here:
https://github.com/zingolabs/zingo-mobile/blob/dev/app/rpc/RPC.ts#L1255
if you havent discussed with juanky might be good before this PR lands

Reply:
Yes, @Oscar-Pepper is right, I thought this was for removing `list`, if you are removing `notes` as well, I need to do same changes in zingo-mobile, and I need some other command or tool instead.

Reply:
Good catch @Oscar-Pepper .